NOTES-NONSTOP.md

```diff
@@ -26,15 +26,14 @@ is the only FLOSS variant that has been broadly tested.
 Threading Models
 ----------------
 
-OpenSSL can be built using unthreaded, POSIX User Threads (PUT), or Standard
-POSIX Threads (SPT). Select the following build configuration for each on
-the TNS/X (L-Series) platform:
+OpenSSL can be built using unthreaded, or POSIX User Threads (PUT). Select
```

Comment:
While we're here, I guess we can fix the grammar:

```suggestion
OpenSSL can be built either using the POSIX User Threads (PUT) threading model, or with threading support disabled. Select
```

Reply:
Done